command/vault/secrets_list.go

```diff
@@ -10,7 +10,7 @@ import (
 
```

Comment:
this is a breaking, unrelated change to your PR title, please open a new PR for discussion on this